internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixReadWriteSchemaRepository.java

```diff
@@ -558,4 +558,10 @@ private void preCheckStoreCondition(String storeName) {
       throw new VeniceNoStoreException(storeName);
     }
   }
+
+  // For testing purpose only.
+  public void removeValueSchema(String storeName, int schemaId) {
```

Comment:
Package private maybe?  This method sounds scary

Reply:
Yeah, a scary api and I was hesitating and trying to find alternative ways of doing it, but I didn't have a good one.

Do we have a convention to put all integration tests under `venice/internal/venice-test-common/src/integrationTest`?  If not, can move it to a different folder. Tried to remove the public keyword with the current folder structure, but compiler complained.

Reply:
If it's only used in an integration test and and not a unit test I think we could put the test case where it would work.  Unit tests are the ones that get finnicky as then they won't attribute coverage properly.

Reply:
Sounds good. I will move it.